docs/frontends/webapi.rst

```diff
@@ -446,14 +446,25 @@ Creating a New Directory
  given, the directory's format is determined by the default mutable file
  format, as configured on the Tahoe-LAFS node responding to the request.
 
+ In addition, an optional "private-key=" argument is supported which, if given,
+ specifies the underlying signing key to be used when creating the directory.
+ This value must be a DER-encoded 2048-bit RSA private key in urlsafe base64
+ encoding. Because this key can be used to derive the write capability for the
+ associated directory, additional care should be taken to ensure that the key is
+ unique, that it is kept confidential, and that it was derived from an
+ appropriate (high-entropy) source of randomness. If this argument is omitted
+ (the default behavior), Tahoe-LAFS will generate an appropriate signing key
+ using the underlying operating system's source of entropy.
```

Comment:
If there's a straightforward command-line to do this, it might be nice to include an example (i.e. to turn a PEM file into "a DER-encoded urlsafe b64" string).

Reply:
I'm not sure if this counts as "straightforward", but [here](https://github.com/tahoe-lafs/tahoe-lafs/pull/1365/commits/692be000a8b467fd50bb38727e92d07db249d481)'s an attempt.